tests/test_text_generator.py

```diff
@@ -0,0 +1,49 @@
+import unittest
```

Comment:
let's make sure the tests are in the same folder structure as the synthetic_data library.
hence, make the `distinct_generators` folder in `tests` and add them there.